src/Assets/TestProjects/VSTestDataCollectorSample/AttachmentProcessorDataCollector.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+	<PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="17.2.0-preview-20220131-20" />
```

Comment:
I don't know if it will be used...`MicrosoftNETTestSdkPackageVersion` is inside /eng/Version.props this directory will be created inside /artifacts/... I need only to be sure to add a version that contains the postprocessing feature and at the moment we have only public preview

Reply:
You're right looks like it's possible we're doing the same in other tests...I'll fix.